core/ui/src/components/software-center/SetInstanceLabelModal.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <NsModal
+    size="default"
+    :visible="visible"
+    @modal-hidden="$emit('hide')"
+    @primary-click="onPrimaryClick"
+  >
```

Comment:
Let’s add `isLoading` prop to show a spinner on the primary button while the changes are being saved